conda/models/match_spec.py

```diff
@@ -433,6 +433,10 @@ def strictness(self):
     def spec(self):
         return self.conda_build_form()
 
+    @property
+    def to_query_param(self):
+        return self.get_exact_value('name')
```

Comment:
currently only passes the package name as a query parameter, like in conda-metachannel. Is it worth passing the entire MatchSpec string representation instead? If so, how likely is the string representation to change? General concern is keeping compatibility with those changes.